bazel-java-deps.bzl

```diff
@@ -119,7 +118,7 @@ def install_java_deps():
             "io.opentelemetry:opentelemetry-context:{}".format(opentelemetry_version),
             "io.opentelemetry:opentelemetry-sdk-testing:{}".format(opentelemetry_version),
             "io.opentelemetry:opentelemetry-sdk-trace:{}".format(opentelemetry_version),
-            "io.opentelemetry:opentelemetry-semconv:{}-alpha".format(opentelemetry_version),
+            "io.opentelemetry:opentelemetry-semconv:1.12.0-alpha",
```

Comment:
This one should probs not be reverted

Reply:
Wasn't this affected in the original PR?

Reply:
Ah yes, it was initially intended to be merged in another PR but was eventually done in this one. You can ignore